pkg/amazon/backend/convert.go

```diff
@@ -37,14 +37,29 @@ func Convert(project *types.Project, service types.ServiceConfig) (*ecs.TaskDefi
 			fmt.Sprintf(" %s.local", project.Name),
 		}))
 
+	tags := []tags.Tag{
+		{
+			Key:   compose.ProjectTag,
+			Value: project.Name,
+		},
+		{
+			Key:   compose.ServiceTag,
+			Value: service.Name,
+		},
+	}
+	tags = append(tags, toTags(service.Labels)...)
+
 	return &ecs.TaskDefinition{
 		ContainerDefinitions: []ecs.TaskDefinition_ContainerDefinition{
 			{
-				Command:               service.Command,
-				DisableNetworking:     service.NetworkMode == "none",
-				DnsSearchDomains:      service.DNSSearch,
-				DnsServers:            service.DNS,
-				DockerLabels:          nil,
+				Command:           service.Command,
+				DisableNetworking: service.NetworkMode == "none",
+				DnsSearchDomains:  service.DNSSearch,
+				DnsServers:        service.DNS,
+				DockerLabels: map[string]string{
```

Comment:
this will create docker engine labels, not available on AWS API
You should have set service.PropagateTags
see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ecs-service.html#cfn-ecs-service-propagatetags